dasharo-compatibility/dasharo-tools-suite.robot

```diff
@@ -83,7 +83,7 @@ DTS005.001 Flash device from DTS shell by using flashrom works correctly
     Boot Dasharo Tools Suite    iPXE
     Enter Shell In DTS
     Set DUT Response Timeout    320s
-    Flash Firmware In DTS
+    Flash Via Internal Programmer
```

Comment:
You need to pass a FW file as a parameter. To do that, you need to download it first, like you do in `ec-and-super-IO.robot`.